packages/svelte2tsx/src/svelte2tsx/nodes/exportsetters.ts

```diff
@@ -0,0 +1,12 @@
+export const createClassSetter = (name: string) =>
+    `\n
+    /**accessor*/\n${' '.repeat(4)}set ${name}(${name}) {}`;
```

Comment:
Could you explain why you add JSDoc here?

Reply:
I though it would be cool to see in the autocomplete list that a prop is an accessor, so users immediately know it can be set / is not an exported readonly const:

![image](https://user-images.githubusercontent.com/3778969/111784835-23283980-88bc-11eb-864f-d4863b646fd4.png)

Reply:
That makes sense, nice quality-of-live improvement 👍